client/lbclient/datasources.json

```diff
@@ -1,6 +1,7 @@
 {
   "remote": {
-    "connector": "remote"
+    "connector": "remote",
+    "url": "/api"
```

Comment:
AFAIK, this value is overriden by [datasources.local.js](https://github.com/strongloop/loopback-example-offline-sync/blob/feature/refactor-with-sync-1.0-changes/client/lbclient/datasources.local.js) and thus not used.